token-plugins-guide.md

```diff
@@ -16,15 +16,15 @@
 
 ## Overview
 
-Token plugins are smart contracts that extend the capabilities of plugin-enabled ERC20 tokens and plugin-enabled ERC20 wrappers. They allow for additional accounting to be added, without the need to deposit your tokens into a contract. **_This is a revolutionary step in the evolution of DeFi_**, as the entire ecosystem has historically relied upon the transfer of tokens in and out of external contracts to achieve enhanced capabilities! 
+Token plugins are smart contracts that extend the capabilities of plugin-enabled ERC20 tokens and plugin-enabled ERC20 wrappers. They allow for additional accounting to be added, without the need to deposit your tokens into a contract. **_This is a revolutionary step in the evolution of DeFi_**, as the entire ecosystem has historically relied upon the transfer of tokens in and out of external contracts to achieve enhanced capabilities! The process of creating and deploying plugins is completely open and permissionless, and holders of plugin-compatible ERC20 tokens can freely participate with any plugin of their choosing.
```

Comment:
Add “without having their assets under risk” at the end

Reply:
Done